ecs_fargate/datadog_checks/ecs_fargate/ecs_fargate.py

```diff
@@ -203,25 +204,34 @@ def submit_perf_metrics(self, container_tags, container_id, container_stats):
 
             cpu_percent = 0.0
             if system_delta > 0 and cpu_delta > 0 and active_cpus > 0:
-                cpu_percent = (cpu_delta / system_delta) * active_cpus * 100.0
-                cpu_percent = round_value(cpu_percent, 2)
-                self.gauge('ecs.fargate.cpu.percent', cpu_percent, tags)
-                if cpu_percent > active_cpus * 100.0:
+                if system_delta > cpu_delta:
+                    cpu_percent = (cpu_delta / system_delta) * active_cpus * 100.0
+                    cpu_percent = round_value(cpu_percent, 2)
+                    self.gauge('ecs.fargate.cpu.percent', cpu_percent, tags)
+                else:
+                    # There is a bug where the system CPU usage is occasionally reported as greater than the CPU usage,
```

Comment:
```suggestion
                    # There is a bug where the system CPU usage is occasionally reported as greater than the system usage,
```

Reply:
`Container CPU Usage is occasionally reported as greater than the system CPU Usage (which, in fact, represents the maximum available CPU time during this timeframe)`